mindsdb_native/libs/phases/data_analyzer/data_analyzer.py

```diff
@@ -281,6 +283,14 @@ def run(self, input_data):
                 stats_v2[col_name]['nr_warnings'] += 1
             self.log.info(f'Finished analyzing column: {col_name} !\n')
```

Comment:
This should come after the final `if`

Reply:
Actually, the whole of adding the text analysis to stats-v2 should come before the above loop that logs the warnings.